databroker/headersource/shim.py

```diff
@@ -24,6 +24,7 @@ def __call__(self, text_search=None, filters=None, **kwargs):
         else:
             # Include empty {} here so that '$and' gets at least one query.
             _format_time(kwargs, self.mds.config['timezone'])
+            [_format_time(v, self.mds.config['timezone']) for v in filters]
```

Comment:
This is still inside the `else` block, meaning that the case under `if` above will be working with un-formatted filters. See my previous comment about moving this line up above the `if` so that it applies to both cases.